module2/lessons/apis.md

```diff
@@ -14,11 +14,25 @@ Then, watch [this video](https://www.youtube.com/watch?v=s7wmiS2mSXY)
 
 ### APIs
 
-At a *very* high level, APIs are rules that applications use to communicate with each other.
+In the broadest sense of the term, an API is the part of an application that other people interact with. Using the example from the video above, restaurant customers don't walk back to the kitchen and give their order to the chef directly. Instead, they give their order to the waiter, the waiter relays that information to the chef, and then the waiter brings the food back to the customer. In this example, the waiter is the API.
 
-We are focused on building Web applications, so we are mostly concerned with **Web Based APIs**. These types of APIs are so common that this is what most people are talking about when referring to an "API".
+You have already used the APIs of many libraries, though up to this point we have not discussed them as such. When you  use RSpec in your test suite, it would technically be accurate to say that you are using the API of the RSpec library. You don't go in and work directly with all of the classes and methods that make up the RSpec library, but instead only the methods included in the documentation that RSpec intends for other people to use. This collection of methods is the RSpec API, which is documented in the [RSpec API documentation](https://rspec.info/documentation/). If you go to that link and click on the latest version of the rspec-core library, you'll see examples that are very familiar to you.
 
```

Comment:
Perhaps a different example might suffice here. While this is focused on a tech students are familiar with students do interact with APIs every day as far as FB, Insta, etc. Or perhaps It may be beneficial to use the example from line 17 as context for line 19 under the premise of what happens when I call "bundle exec RSpec".